benchmark/reformat_input.py

```diff
@@ -113,8 +113,8 @@ def species_gene_mapping(gene_tree,
         else:
             raise ValueError(
                 "Species %s does not exist in the species tree" % species)
-
-    return mapping_leaves
+    return OrderedDict(sorted(mapping_leaves.items(),
```

Comment:
The docstring indicates a `list` `tuple`s, but this is an `OrderedDict`. Would it be possible to clarify the difference? Second, out of curiosity, what's the motivation for an `OrderedDict`?